tornado-api/src/main/java/uk/ac/manchester/tornado/api/collections/types/ImageFloat4.java

```diff
@@ -167,7 +167,7 @@ public String toString(String fmt) {
     public String toString() {
         String result = String.format("ImageFloat4 <%d x %d>", X, Y);
         if (X <= 8 && Y <= 8) {
-            result += "\n" + toString(FloatOps.fmt3);
+            result += "\n" + toString(FloatOps.FMT_3);
```

Comment:
Should not be this `FloatOps.FMT_4`?

Reply:
@mikepapadim , when you get the time, please take a look at this and then we can merge.

Thank you!